client/src/modules/admin/index.js

```diff
@@ -0,0 +1,5 @@
+import AdminList from "./components";
+
+export {
```

Comment:
write in single line! why did you use 3 lines to do a single line export! goddamnit!